context_test.go

```diff
@@ -14,3 +14,16 @@ func TestContext(t *testing.T) {
 	assert.Equal(t, httpCtx.Value("Hello").(string), "world")
 	assert.Equal(t, httpCtx.Value("Hi").(string), "Goravel")
 }
+
+func TestContextWithCustomKeyType(t *testing.T) {
+	type customKeyType struct{}
+	var customKey customKeyType
+	var customKeyTwo customKeyType
```

Comment:
Please add additional other type keys: int, string, etc.

Reply:
We may merge these two test cases: `TestContext`, `TestContextWithCustomKeyType`.

Reply:
did you mean one function Test but with many sub-test @hwbrzzl ?

Reply:
Yes, the test cases are very simple, two functions are unnecessary.

Reply:
done in https://github.com/goravel/fiber/pull/102/commits/9989eb4ce2ee4875e23cb85080f647f2f038b05e